### ISSUE TITLE
Connection to Extensions Marketplace fails.

### ISSUE THREAD

**Issue by Moghi** [VSCodium/vscodium#783]
**Describe the bug**
Cannot connect to the Extensions Marketplace.

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Go to 'Extensions Tab'
2. Search for any extension

**Expected behavior**
Showing extensions in the marketplace.

**Desktop:**
 - OS: Mac OS
 - Architecture x64
 - Version 1.58.2

**Errors shown in developer tools console**
in `workbench.html:1`:
```
Access to XMLHttpRequest at 'https://open-vsx.org/vscode/gallery/extensionquery' from origin
'vscode-file://vscode-app' has been blocked by CORS policy: Response to preflight request doesn't pass
access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

in `extensionsViews.ts:323`:
```
Error querying extensions gallery XHR failed
```

in `request.ts:42`:
```
POST https://open-vsx.org/vscode/gallery/extensionquery net::ERR_FAILED
```

**Details of the failed request**
![Screenshot 2021-08-02 at 11 46 44](https://user-images.githubusercontent.com/12676705/127841469-250e723f-29db-4c61-9df4-beb0d6536f59.png)

**Comment by daiyam** [comment]
Hi, it's a duplicate of #746.
